Palace/Accounts/Library/Account.swift

```diff
@@ -138,7 +138,7 @@ class OPDS2SamlIDP: NSObject, Codable {
     
     var catalogRequiresAuthentication: Bool {
       // you need an oauth token in order to access catalogs if authentication type is either oauth with intermediary (ex. Clever), or SAML
-      authType == .oauthIntermediary || authType == .saml
```

Comment:
That's scary! I re-reviewed recent PRs but couldn't find when it was removed. Thanks for noticing that!

Reply:
David caught it, I did it when I was reverting a poor merge conflict resolution on the refactor branch.